index.html

```diff
@@ -269,21 +269,34 @@ <h3>LET'S CONNECT</h3>
 						/>
 						<ul class="about__main-content--skills--list">
 							<li>
-								<img src="assets/img/js.svg" alt="database" />
+								<img src="assets/img/JS.svg" alt="database" />
 								Database Management
 							</li>
 							<li>
-								<img src="assets/img/js.svg" alt="github" />
+								<img src="assets/img/JS.svg" alt="github" />
 								Version Control
 							</li>
 							<li>
-								<img src="assets/img/js.svg" alt="web development" />
+								<img src="assets/img/JS.svg" alt="web development" />
 								Web Development
 							</li>
 						</ul>
 					</li>
 				</ul>
 			</div>
 		</section>
+
+		<section class="contact">
+			<h2 class="contact__title">Contact me</h2>
+			<p class="contact__description">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+				I’d love to help with it!</p>
+			<form action="https://formspree.io/f/mwkjredy" method="POST" class="contact__form">
+				<input type="text" name="name" placeholder="Name" value="Lorem Ipsum" maxlength="30" required>
+				<input type="email" name="email" placeholder="E-mail" value="loremipsum@example.com" id="email" required>
```

Comment:
- [Optional] I suggest removing the value attribute for all your input fields. This will be better for user experience. Currently, the input fields are prefilled, and the user must clear them before they can add their data.